src/executables/utility.h

```diff
@@ -1497,6 +1498,7 @@ typedef struct _ha_config
 #define APPLYINFO_VERBOSE_L                     "verbose"
 #define APPLYINFO_INTERVAL_S                    'i'
 #define APPLYINFO_INTERVAL_L                    "interval"
+#define APPLYINFO_NULL_LOG_PAGEID               LOGPAGEID_MAX+1
```

Comment:
I think this name may be confused with the existing NULL_LOG_PAGEID which is -1. 

It seems to be to properly check the value entered by the user by setting invalid values ​​as default values, is that correct? 

Then, how about `APPLYINFO_INVALID_LOG_PAGEID` or `APPLYINFO_DEFAULT_LOG_PAGEID` ?

Reply:
Yes. the value was -1 and it can be the input as the entering value -1 by a user.

Thank you. APPLYINFO_DEFAULT_LOG_PAGEID will be better. I will apply it to the code.